cmd/clusterctl/cmd/root.go

```diff
@@ -19,6 +19,7 @@ package cmd
 import (
 	"flag"
 	"fmt"
+	"k8s.io/klog"
```

Comment:
Keep this in the second set of imports rather than grouping with the stdlib imports.